PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,33 @@
-**Please check the type of change your PR introduces:**
-- [ ] Bugfix
-- [ ] Feature
-- [ ] Refactoring / Code style update (no functional changes, no api changes)
-- [ ] Build related changes
-- [ ] Documentation
-- [ ] Other (please describe): 
+### Describe the type of change your PR introduces:
 
+<!-- REMOVE ALL THAT DON'T APPLY -->
```

Comment:
When I open a PR, I don't like to remove things from a template. And I think that this is similar to the one removed (with emoji instead of checkboxes.
Since below there is a section "Problem or request" this is redundant

Reply:
Good point! 
How could we express a "PR category" to be expressed though?
You are right that's redundant, maybe I just want to know if it's a CHANGE/FIX/REFACTOR/FEATURE, I could add it as a comment?

Reply:
Moved a shorter text in the Solution section